create.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Create</title>
+  </head>
+
+  <body>
+    <header>
+      <h1>CREATE</h1>
+    </header>
+
+    <main>
+      <section class="text--field">
+        <lable for="text--field__question">Creat a tricky question?:</lable>
+        <textarea
+          id="text--field__question"
+          name="text--field__question"
+          rows="5"
+          cols="33"
+        >
+How many seconds are there in a year?</textarea
+        >
+
+        <lable for="text--field__answer">Give a wise answer!:</lable>
+        <textarea
+          id="text--field__answer"
+          name="text--field__answer"
+          rows="5"
+          cols="33"
+        >
+Twelve. January 2nd, February 2nd, March 2nd, etc.</textarea
+        >
+
+        <lable for="text--field__tags">Add tags divided by comma:</lable>
+        <textarea
+          id="text--field__tags"
+          name="text--field__tags"
+          rows="5"
+          cols="33"
+        >
+#,</textarea
+        >
+        <input type="submmit" value="SUBMIT" />
```

Comment:
I would recommend you to use a button here instead of an input field. As well, as you have learned it this morning, I suggest to wrap your textareas and the submit-button with the `<form>`-element
```suggestion
        <button>Submit</button>
```